registration-system/admin/pages_mail.php

```diff
@@ -154,7 +157,12 @@ private function buildQueryWhere() {
                 $where['abtyp'] = $_REQUEST['val_abtyp'];
             }
             if (isset($_REQUEST['check_nights'])) {
-                // TODO
+                $nights = $_REQUEST['val_nights'];
+                $conditions = [];
+                foreach ($nights as $night)
+                    $conditions['AND #'.$night] = ['anday[<=]' => $night, 'abday[>]' => $night];
+                if (sizeof($conditions) > 0)
+                    $where['OR #nights'] = $conditions;
```

Comment:
warum ist hier ein `OR` und bei allen anderen keys in `$where` nicht?

Reply:
wegen 'in mindestens einer selektierten nacht anwesend' logik

Reply:
Ja, das stimmt schon, bin mir nur nicht sicher, ob man lustige Seiteneffekte bekommt (Klammern/Scopes).

Beispiel: [[bezahlt] UND [auto] ODER [nacht1] ODER [nacht1 UND nacht2]]
Wäre ja doof dann, weil die beiden anderen (bezahlt, auto) irrelevant werden.

Reply:
die oberste ebene ist via _und_ verknüpft - darunter sind die einzelnen nächte via oder verknüpft. also hat keine aushebelnde wirkung auf die bisherige selektion